pom.xml

```diff
@@ -18,6 +18,7 @@
         <module>service</module>
         <module>iot-e2e-tests</module>
         <module>provisioning</module>
+        <module>Quickstarts/read-d2c-messages</module>
```

Comment:
This module's parent should not be the root pom.xml file. It's parent should be the [iot-service-samples pom](https://github.com/Azure/azure-iot-sdk-java/blob/main/service/iot-service-samples/pom.xml) since the sample will live alongside the service client samples

Reply:
This is weird... I changed this before your latest reviews, so the module `read-d2c-messages` shouldn't be in the root pom anymore. Can you still see the changes on root pom [here](https://github.com/Azure/azure-iot-sdk-java/pull/1597/files)?